internal/errgroup/group_test.go

```diff
@@ -729,49 +738,34 @@ func Test_group_Wait(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       fields: fields {
-		           egctx: nil,
-		           cancel: nil,
-		           wg: sync.WaitGroup{},
-		           limitation: nil,
-		           enableLimitation: nil,
-		           cancelOnce: sync.Once{},
-		           mu: sync.RWMutex{},
-		           emap: nil,
-		           errs: nil,
-		           err: nil,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           fields: fields {
-		           egctx: nil,
-		           cancel: nil,
-		           wg: sync.WaitGroup{},
-		           limitation: nil,
-		           enableLimitation: nil,
-		           cancelOnce: sync.Once{},
-		           mu: sync.RWMutex{},
-		           emap: nil,
-		           errs: nil,
-		           err: nil,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
```

Comment:
do we need some test case to block the current `Go(func())` call and the `Wait()` should actually wait for all functions return?